bazel-java-deps.bzl

```diff
@@ -44,7 +44,7 @@ netty_version = "4.1.67.Final"
 # <tcnative.version>2.0.42.Final</tcnative.version>
 # ```
 netty_tcnative_version = "2.0.40.Final"
-grpc_version = "1.41.0"
+grpc_version = "1.35.0"
```

Comment:
See https://github.com/digital-asset/daml/pull/11748/files#r751503832, this just reverts stuff back to what we were already using.